test/tools/Modules/PSSysLog/PSSysLog.psm1

```diff
@@ -406,12 +406,28 @@ class PSLogItem
                 $osLogIds.UseOldIds()
             }
 
-            $item.Message = $parts[$osLogIds.Message]
+            if($parts.Length -gt ($osLogIds.Message +1))
+            {
+                # The message got split into multiple parts, put it back together
+                $item.Message = $parts[$osLogIds.Message..($parts.Length-1)] -join ' '
+            }
+            else
+            {
+                $item.Message = $parts[$osLogIds.Message] -join ' '
+            }
 
             # [com.microsoft.powershell.logid]
             $splitChars = ('[', '.', ']')
 
             $item.LogId = $parts[$osLogIds.Id]
+
+            if($item.LogId -notmatch '^\[com\.microsoft\.powershell')
+            {
+                Write-Verbose "logId: $($item.LogId)" -Verbose
```

Comment:
Suggest changing the verbose message slightly, to something like:

"Skipping logId: $($item.LogId)"